.travis.yml

```diff
@@ -1,5 +1,17 @@
+language: python
+python:
+  - "3.8"
 services:
   - docker
-
+  - postgresql
+env:
+  - DATABASE_URL="postgres://postgres:postgres@127.0.0.1:5432/securemailbox"
+install:
+  - pip install --upgrade pip setuptools
```

Comment:
Nit: The `install pipenv` can be done here if you want. 
```bash
pip install --upgrade pip setuptools pipenv
```

Reply:
Yes that makes sense. I'll change it.